src/components/auction/OrderPlacement/index.tsx

```diff
@@ -170,6 +170,9 @@ const OrderPlacement: React.FC<OrderPlacementProps> = (props) => {
   const [attemptingTxn, setAttemptingTxn] = useState<boolean>(false) // clicked confirmed
   const [pendingConfirmation, setPendingConfirmation] = useState<boolean>(true) // waiting for user confirmation
   const [txHash, setTxHash] = useState<string>('')
+  const [hasRiskNotCoveringClearingPrice, setHasRiskNotCoveringClearingPrice] = useState<boolean>(
```

Comment:
Why is it needed to introduce a new state ? Why can't we just declare the variable `hasRiskNotCoveringClearingPrice` and assign it?

Reply:
I totally agree with you 👌!